design-documents/service-isolation/cicd-pipeline.md

```diff
@@ -0,0 +1,48 @@
+# CI/CD pipeline for service isolation
+
+## About
+
+In the scope of service isolation we want to have CI/CD process that would allow
+* Running tests on change set and update preview environment
```

Comment:
We might consider something like [Netlify](https://www.netlify.com/) to automatically create these preview environments for each pull request.